docs/support.md

```diff
@@ -10,7 +10,7 @@ If you have any problems or questions about working with Wagtail, you are invite
 
 ## Mailing list
 
-For topics and discussions that do not fit Stack Overflow's question-and-answer format, there is a Wagtail Support mailing list at [groups.google.com/d/forum/wagtail](https://groups.google.com/d/forum/wagtail).
+For topics and discussions that do not fit Stack Overflow's question-and-answer format, there is a Wagtail Support mailing list at [groups.google.com/d/forum/wagtail](https://groups.google.com/g/wagtail).
```

Comment:
suggestion: let's link to GitHub Discussions

Reply:
yep - will update this one, to align with the readme.

Reply:
Actually we refer to both slack and the GitHub discussions on this same page further down.

I might leave this as is then until we get a call on whether we should remove references to the Google Group completely.